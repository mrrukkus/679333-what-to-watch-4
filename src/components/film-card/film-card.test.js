import React from "react";
import renderer from "react-test-renderer";
import FilmCard from "../film-card/film-card.jsx";

it(`Film cards renders correctly`, () => {
  const card = renderer
    .create(<FilmCard
      filmTitle={`Fantastic Beasts: The Crimes of Grindelwald`}
      filmImageLink={`img/fantastic-beasts-the-crimes-of-grindelwald.jpg`}
      onCardAction={() => {}}
    />).toJSON();

  expect(card).toMatchSnapshot();
});
