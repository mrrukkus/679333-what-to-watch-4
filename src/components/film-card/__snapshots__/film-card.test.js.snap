// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Film card renders correctly 1`] = `
<a
  className="small-movie-card catalog__movies-card small-movie-card__link"
  href="/films/undefined"
  onClick={[Function]}
  onMouseOut={[Function]}
  onMouseOver={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <img
      alt="Fantastic Beasts: The Crimes of Grindelwald"
      height="175"
      src="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <div
      className="small-movie-card__link"
    >
      Fantastic Beasts: The Crimes of Grindelwald
    </div>
  </h3>
</a>
`;
